openlibrary/plugins/openlibrary/js/compact-title/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Defines functions related to the compact title component.
+ * @module compact-title/index
+ */
+
+/**
+ * True if compact title component is visible on screen.
+ * @type {boolean}
+ */
+let isTitleVisible = false
+
+/**
+ * Navbar is "stuck" when it reaches this position on the Y-axis.
+ */
+const navbarStickyHeight = getTitleComponentHeight()
+
+/**
+ * Returns a number representing the compact title component's height.
+ *
+ * The compact title's height is fetched from a CSS variable.
+ *
+ * @returns {Number} The height of the compact title component.
+ */
+function getTitleComponentHeight() {
+    const titleHeightWithUnits =  getComputedStyle(document.body).getPropertyValue('--compact-title-height');
+
+    return Number(titleHeightWithUnits.replace(/\D/g,''))
```

Comment:
`parseFloat` ignores trailing strings :) 

```suggestion
    return parseFloat(titleHeightWithUnits)
```